### PR TITLE
Add automated tests

### PR DESCRIPTION
**Pull request by sravfeyn**

### CODE REVIEW

File: hq_superset/tests/config_for_tests.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            'access_token_url': 'http://127.0.0.1:8000/oauth/token/',
+            'authorize_url': 'http://127.0.0.1:8000/oauth/authorize/',
+            'client_kwargs': {
+                'scope': 'reports:view access_apis'
+            },
+        }
+    }
+]
+
+# This is set by
```
Comment (kaapstorm) [inline hq_superset/tests/config_for_tests.py:37]: ?

**Review by kaapstorm — APPROVED** [review]
This is great.

**Review by Charl1996 — APPROVED** [review]

File: hq_superset/tests/config_for_tests.py (hunk 1 above)
Comment (sravfeyn) [reply hq_superset/tests/config_for_tests.py:37]: Probably an old partially edited comment, removing it.

File: .gitignore [hunk 2]
```diff
@@ -115,3 +115,4 @@ tests/.superset/
 *.sublime-project
 *.sublime-workspace
 superset_config.py
+*superset.db
```
Comment (Charl1996) [inline .gitignore:118]: Nit: needs blank line

Comment (sravfeyn) [reply .gitignore]: https://github.com/dimagi/hq_superset/pull/22/commits/1e43a38431a555e3ec28d46bd47116c578b8abb6

**Review by Charl1996 — APPROVED** [review]

**Pull request merged**